llvm/lib/Analysis/TargetLibraryInfo.cpp

```diff
@@ -852,6 +852,7 @@ static void initializeLibCalls(TargetLibraryInfoImpl &TLI, const Triple &T,
     TLI.setUnavailable(LibFunc_memrchr);
     TLI.setUnavailable(LibFunc_ntohl);
     TLI.setUnavailable(LibFunc_ntohs);
+    TLI.setUnavailable(LibFunc_reallocarray);
```

Comment:
"Since 2.29 under _DEFAULT_SOURCE and under _GNU_SOURCE"

Can you check the version and comment this somewhere?

Reply:
That's from the manpage (e.g. https://man.archlinux.org/man/reallocarray.3.en). I'll update the commit message to make it clear.

Reply:
I mean this needs to be documented in the code, not the commit message

Reply:
done!

Reply:
How did you determine whether this is available on PS?

Reply:
I grepped the whole sdk (for PS4 though). Plenty of mention to `realloc` and none to `reallocarray`